lte/protos/BUILD.bazel

```diff
@@ -454,3 +454,32 @@ python_grpc_library(
     name = "subscriberauth_python_grpc",
     protos = [":subscriberauth_proto"],
 )
+
+# This proto library is exclusively meant for the state service.
+# Users can manually add proto libraries to state.yml.
+# This means that all proto libraries need to be available
+# in builds of the state service.
+proto_library(
+    name = "all_proto",
+    srcs = glob(["*.proto"]),
```

Comment:
My first thought was: `Generally in Bazel globbing of source is an anti-pattern (though OK if you are rapidly standing something up - generally needs to be refactored to remove globbing` - but then with Sourcegraph search I found [171 instances of this type of globbing for proots](https://sourcegraph.com/search?q=context:global+srcs+%3D+glob%28%5B%22*.proto%22%5D%29%2C&patternType=literal) including some Google repos, so that seems OK.  I guess what still isn't confirmed and feels strange to me is to have more than one Bazel label referencing the same source (this feels a bit anti-DRY).  I do appreciate that you put the comment here so that others don't lazily include this target if un-necessary.

The RISK IMO is that people lazily use all_proto and then have much larger than necessary build graphs when iterating simple unit tests on targets needing only one or two protos.

Aaah And i see you have dealt with this risk by making visibility of all_python_proto only visible to `python/magma/state:__pkg__`.

So this all seems reasonable to me =D

Reply:
Edit: Can you imagine dealing with this cleanly and safely like this in lesser build systems? Toss this on the Bazel-win pile.

Reply:
> Aaah And i see you have dealt with this risk by making visibility of all_python_proto only visible to `python/magma/state:__pkg__`.

:heavy_plus_sign: I have to agree, that this idea is quite good! :rocket: